SingularityUI/app/views/tail.coffee

```diff
@@ -57,10 +62,14 @@ class TailView extends View
         # Well, or just render them if we're starting fresh
         $firstLine = @$linesWrapper.find '.line:first-child'
         $lastLine  = @$linesWrapper.find '.line:last-child'
+        url = window.location.host + window.location.pathname
```

Comment:
i dont think this is necessary for `@linesTemplate` -- `{{url}}` isnt in the template anywhere